spec/jobs/activity_tracking_job_spec.rb

```diff
@@ -60,20 +60,23 @@
     end
 
     it 'creates notifications when a comment is created' do
-      commentable = create(:issue)
-      create_list(:subscription, 2, subscribable: commentable)
-      trackable = create(:comment, commentable: commentable)
-      project = commentable.node.project
+      commentables = [:issue, :note]
```

Comment:
I'm not sure about this code, the specs seems to be trying to do too much?

Maybe split in different scopes, or into a shared example or something else?

Otherwise by the end of the cycle we're going to have a mammoth spec for :issue, :evidence, :note, :card, etc.